### ISSUE TITLE
Services getting constructed whenever page is shown

### ISSUE THREAD

**Issue by mdrillin** [teiid/beetle-studio#37]
The ActivityService and ConnectionService appear to get constructed every time the summary page is shown.  Needs further investigation.

**Comment by elvisisking** [comment]
In the summary page components, don't add the services as providers. This causes a new instance of the service to be constructed each time a summary page is constructed.

So remove this: 
```
providers: [ ActivityService ]
``` 
from `ActivitiesComponent`, and remove this:
```
providers: [ConnectionService]
``` 
from `ConnectionsComponent`.

**Comment by mdrillin** [comment]
This was resolved with PR #41
